modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -179,7 +179,7 @@ NOTE: Activating this option means any user authorized by the IdP to access Boni
  ** Some OpenID Connect providers do not support having a pattern of valid redirect URIs. It is the case with Google's for example. With those providers you can use the `state` parameter to pass the target URL (the URL you initially requested and want to be redirected to). In order to do that set the property `save-target-url-in-state` to `true`. When you do that, the redirect_URI query parameter will always be the same. Its default value is the path `/bonita/keycloak` on Bonita server, but you can set a different value using the property `redirect-uri-path`. Make sure this URL is part of the valid redirect URIs in you OIDC provider configuration for the Bonita client. When `save-target-url-in-state` is set to `false` or not present, the `redirect_URI` query parameter is used to save the target URL. Those properties are specific to Bonita OIDC module and are not supported/documented by Keycloak.
- ** the `ssl-required` property value may need to be changed if Bonita Portal and the IdP are not both accessed via HTTPS. Possible values for this property are: `all`, `external`, and `none`. For `all`, all requests must come in via HTTPS. For `external`, only non-private IP addresses must come over via HTTPS. For `none`, no requests are required to come over via HTTPS.
+ ** the `ssl-required` property value may need to be changed if Bonita Runtime and the IdP are not both accessed via HTTPS. Possible values for this property are: `all`, `external`, and `none`. For `all`, all requests must come in via HTTPS. For `external`, only non-private IP addresses must come over via HTTPS. For `none`, no requests are required to come over via HTTPS.
  ** the `principal-attribute` value indicates the OIDC ID Token attribute/claim (`preferred_username`, `sub`, `email`, `name`, etc...) to use to be considered as user identifier in the client application. It should match the username in Bonita.
  ** `public-client` needs to be set to `false` if the OIDC provider requires the client requests to be authenticated (with a client secret for example). In this case you also need to use the `credentials` property. If this property is set to `true`, then no client Authentication header will be sent to the OIDC provider.
  ** `enable-cors` is required only if you wan to be able to use Bonita REST API from a web application located on a different domain from your Bonita installation's. It comes with additional options `cors-max-age`, `cors-allowed-methods`, `cors-allowed-headers`, `cors-exposed-headers`. You can check Keycloak https://www.keycloak.org/docs/latest/securing_apps/#_java_adapter_config[client adpter config documentation] for more information.
```

Comment:
`enable-cors` is required only if you wan => `enable-cors` is required only if you want

Reply:
Added